rclcpp/include/rclcpp/service.hpp

```diff
@@ -486,6 +486,14 @@ class Service
   {
     rcl_ret_t ret = rcl_send_response(get_service_handle().get(), &req_id, &response);
 
+    if (ret == RCL_RET_TIMEOUT) {
+      RCLCPP_WARN(
+        rclcpp::get_node_logger(node_handle_.get()).get_child("rclcpp"),
+        "failed to send response (timeout): %s",
```

Comment:
I suggest to add the service name in the log message

Reply:
I think the problem with trying to do that here is that not all of the constructors have access to the service name.  So this may not be possible to do without changing all of the constructors, or exposing `service_name` in `rcl_service_t`.  Personally, I think this is nice but I wouldn't hold up this PR for that.